src/main/java/seedu/address/logic/commands/ListTodayPrescriptionCommand.java

```diff
@@ -0,0 +1,69 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.util.function.Predicate;
+
+import javafx.collections.ObservableList;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.ModelPrescription;
+import seedu.address.model.prescription.Frequency;
+import seedu.address.model.prescription.Prescription;
+
+/**
+ * Lists all remaining medications to be taken for the day to the user.
+ */
+public class ListTodayPrescriptionCommand extends CommandPrescription {
+
+    public static final String COMMAND_WORD = "listToday";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Lists all medications to be taken today.\n"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String MESSAGE_SUCCESS = "Listing all medications to be taken today";
+    public static final String MESSAGE_EMPTY_LIST = "No medications to be taken today";
+
+    @Override
+    public CommandResult execute(ModelPrescription model) throws CommandException {
+
+        requireNonNull(model);
+
+        Predicate<Prescription> isToday = prescription -> {
+            LocalDate today = LocalDate.now();
+            LocalDate startDate = LocalDate.parse(
+                    prescription.getStartDate().fullDate, DateTimeFormatter.ofPattern("dd/MM/yyyy"));
```

Comment:
For this, can change ur getDate method to just return the fullDate attribute instead of parsing it into a date object! Cuz from what i see, getDate was not used elsewhere?
Then fullDate can be private